samples/MvcSandbox/Startup.cs

```diff
@@ -15,19 +17,26 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(Microsoft.AspNetCore.Mvc.CompatibilityVersion.Version_2_2);
+
+
+            services.Configure<MvcEndpointDataSourceOptions>(o =>
+            {
+                o.Endpoints.Add(new EndpointInfo()
+                {
+                    Template = "{controller=Home}/{action=Index}/{id?}",
+                    Name = "default"
+                });
+            });
+
```

Comment:
Discussed this in person - this was just done to verify the E2E, he's not planning to check it in since @JamesNK is doing this work right now.